clusterloader2/pkg/provider/provider.go

```diff
@@ -33,6 +33,9 @@ type InitOptions struct {
 
 // Features represents all features supported by this provider.
 type Features struct {
+	// KubeStateMetricsTestDisabled determines if running kube-state-metrics is supported.
+	KubeStateMetricsTestDisabled bool
```

Comment:
We're trying to be positive in those (e.g. talk about features not about disabling something).

So instead of that, we should make it more like:

`SupportKubeStateMetrics`

Given that our tests run on GCE, it's probably enough to enable that just for gce cloud provider for now.